src/webview/markdownEngine.ts

```diff
@@ -0,0 +1,99 @@
+import * as hljs from "highlight.js";
```

Comment:
file name change to `MarkdownEngine` since it's the only class exported in this file.

Also, remember to correct the import statement to avoid compilation error